tools/node_compat/deno.json

```diff
@@ -0,0 +1,6 @@
+{
+  "tasks": {
+    "setup": "deno run --allow-read --allow-write ./setup.ts",
+    "test": "deno test -A ../../cli/tests/node_compat/test.ts -- $TARGET"
```

Comment:
Why have a variable? Couldn't someone just do `deno task test buffer`?

Reply:
It becomes simple and fix it. https://github.com/denoland/deno/pull/19293/commits/d2d0ca9fa4569199557409cb1bd1782cb6a4a06b